index.js

```diff
@@ -114,9 +90,16 @@ function FastForEach(spec) {
   this.afterQueueFlush = spec.afterQueueFlush;
   this.beforeQueueFlush = spec.beforeQueueFlush;
   this.changeQueue = [];
-  this.lastNodesList = [];
+  this.firstLastNodesList = [];
   this.indexesToDelete = [];
   this.rendering_queued = false;
+  this.pendingDeletes = [];
+  // for testability purposes only
+  if (FastForEach.DEBUG) {
+    this.manualProcessChangeQueue = spec.manualProcessChangeQueue;
```

Comment:
This is the same as setting `FastForEach.animateFrame` to a no-op function (`function (){}`), which is probably preferable because it doesn't clutter the code with debug stuff. :)